rs/ethereum/ledger-suite-orchestrator/ledger_suite_orchestrator.did

```diff
@@ -44,6 +44,11 @@ type UpgradeArg = record {
 
    // Update the cycles management of the canisters managed by the orchestrator.
    cycles_management: opt UpdateCyclesManagement;
+
+   // Add already installed canisters to the canisters managed by the orchestrator.
+   // Those canisters are *NOT* ckERC20 tokens.
+   // This assumes that the orchestrator is a controller of all the canisters in the list.
+   manage_other_canisters: opt vec ManageOtherCanisters;
```

Comment:
For the sake of consistency, how about calling it `OtherManagedCanisters` (in line with `ManagedCanisters`)?

Reply:
I'm also note that that satisfied with this naming. Maybe let's keep the naming discussion to a single thread, e.g. [here](https://github.com/dfinity/ic/pull/1312#discussion_r1749987619).

Reply:
Sounds good.